docs/docs/Flows/lfx.mdx

```diff
@@ -0,0 +1,357 @@
+---
+title: Run flows with Langflow Executor (LFX)
+slug: /lfx-stateless-flows
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import PartialLfxDependencies from '@site/docs/_partial-lfx-dependencies.mdx';
+
+The Langflow Executor (LFX) is a command-line tool that serves and runs flows statelessly from [flow JSON files](/concepts-flows-import) with minimal dependencies.
+
+Running a flow with LFX is similar to running flows with the [`--backend-only` environment variable](/environment-variables#server) enabled, but even more lightweight because the Langflow package and all of its dependencies don't need to be installed.
+
+Use LFX to share flows with other developers, test flows in different environments, and run flows in production applications without requiring the full Langflow UI or database setup.
+
+LFX includes two commands for executing flows:
+
+* [`lfx serve`](#serve): This command starts a FastAPI server hosting a Langflow API endpoint with your flow available at `/flows/{flow_id}/run`.
+The flow graph is stored in memory at all times, so there is less overhead for loading the graph from a database.
+* [`lfx run`](#run): This command executes a flow locally and returns the results to `stdout`.
+
+## Prerequisites
+
+- Install [Python](https://www.python.org/downloads/release/python-3100/).
+- Install [uv](https://docs.astral.sh/uv/getting-started/installation/).
+- Create or download a [flow JSON file](/concepts-flows). For example, download the Simple Agent flow from the repository:
+  ```bash
+  curl -o simple-agent-flow.json "https://raw.githubusercontent.com/langflow-ai/langflow/main/src/backend/base/langflow/initial_setup/starter_projects/Simple%20Agent.json"
+  ```
+- Create an [OpenAI API key](https://platform.openai.com/api-keys).
+- Create a Langflow API key. For LFX, you can [generate a secure token locally](#serve), or create one through the [Langflow server UI or CLI](/api-keys-and-authentication).
```

Comment:
nit: I wouldn't point them to Langflow UI because it may confuse them (they may expect that just by creating it will be set in the env).
```suggestion
- Create a Langflow API key. For LFX, you can [generate a secure token locally](#serve).
```